katana_requirements.yaml

```diff
@@ -373,7 +373,7 @@ python:
     conan: null
     apt: python3
   # TODO(amp): Unpin this. Currently we only build for python 3.8 so pin to that to reduce confusion.
-  version: [ 3.8, 3.9 ]
+  version: [ 3.8, 3.9.6 ]
```

Comment:
@arthurp I had to bump the allowed versions of python & ccache for 21.04, is that a problem for our standard 20.04?

Reply:
YES!!! Nothing builds on python 3.8 yet. So if you want to do this, we need to upgrade everything to build on newer python.

ccache upgrade is just fine.

Reply:
BTW, I really do want to upgrade to supporting 3.8, 3.9, AND 3.10. But I haven't had time.

Reply:
hm, then full 21.04 support will have to wait for now as keeping python pinned as-is causes apt package conflicts during script run-time. 
All of my builds seem happy enough on 3.9.5, but maybe I'm not touching stuff that would break.
I'll revert the python version change for now.

Reply:
Well you should be able to run everything in conda just fine. Using APT packages is getting deprecated anyway.

I think rolling this back for the moment is safest.

Reply:
Yeah, I'm one of the few holdouts for now. I really should switch over to conda, but I have yet to find the time to debug the issues I had with it last time I tried.

Reply:
I understand.

If you can reproduce your problems in a docker container or on a machine I can log into, I would be happy to debug it and fix the tooling. When I can time, which might be a bit. Tooling bugs are bugs. Even if they only happen in some environments.